vck/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/VerifierAgent.kt

```diff
@@ -53,7 +54,7 @@ class VerifierAgent(
             ?.let { bytes -> Document.deserialize(bytes).getOrNull() }
         if (document != null) {
             val verifiedDocument = runCatching {
-                validator.verifyDocument(document, challenge)
+                validator.verifyDocument(document, { _, _ -> true })
```

Comment:
while I have no doubt that this is correct, it *looks* funky. (same goes for line 67).

Reply:
Implemented a fallback for this deprecated (and internally never called method) in 693c3570